models/user.js

```diff
@@ -17,6 +17,7 @@ module.exports = (sequelize, DataTypes) => {
      */
     static associate(models) {
       // define association here
+      User.hasOne(models.Budget);
```

Comment:
Here, in User we set up the other side of the relationship. The User HAS ONE Budget, making this a ONE TO ONE relationship...